RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3014,7 +3014,7 @@ boolean auto_is_valid(familiar fam)
 	{
 		return to_familiar(get_property("auto_100familiar")) == fam;
 	}
-	return bhy_usable(fam.to_string()) && glover_usable(fam.to_string()) && zombieSlayer_usable(fam) && is_unrestricted(fam);
+	return bhy_usable(fam.to_string()) && glover_usable(fam.to_string()) && zombieSlayer_usable(fam) && is_unrestricted(fam) || (in_wereprof() && fam != $familiar[stooper]);
```

Comment:
My preference would be to make a wereprof_usable function to be consistent with other paths here